l10n_it_reverse_charge/models/account_invoice.py

```diff
@@ -35,22 +34,43 @@ class AccountInvoice(models.Model):
         comodel_name='account.invoice',
         string='RC Self Purchase Invoice', copy=False, readonly=True)
 
+    rc_partner_supplier_id = fields.Many2one(
```

Comment:
This is new functionality, please implement new functionalities out of the migration commit, thanks